srcalc/src/main/java/gov/va/med/srcalc/util/MissingValueListException.java

```diff
@@ -0,0 +1,26 @@
+package gov.va.med.srcalc.util;
+
+import java.util.List;
+
+import gov.va.med.srcalc.domain.variable.MissingValueException;
+
+/**
+ * Indicates that there is at least one {@link MissingValueException}
+ */
+public class MissingValueListException extends Exception
```

Comment:
I prefer `MissingValuesException` to avoid specifying that it's a List... could be a Set later. But if you don't want to change the name, that's OK.